### PR TITLE
Add support for SeaHorn.

### PR DESCRIPTION
**Pull request by fshaked**
Missing documentation, and integration with cargo-verify.

### CODE REVIEW

File: verification-annotations/lib/seahorn.c [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#include <stdlib.h>
+#include <stdint.h>
+
+void __VERIFIER_error() {}
```
Comment (alastairreid) [inline verification-annotations/lib/seahorn.c]: I would have expected __VERIFIER_error to call a function to indicate that an error occurred?
(eg maybe "abort" or something?)

File: verification-annotations/src/seahorn.rs [hunk 2]
```diff
@@ -166,6 +196,13 @@ macro_rules! assert_ne {
     }};
 }
 
+#[macro_export]
```
Comment (alastairreid) [inline verification-annotations/src/seahorn.rs]: This is not provided by KLEE. Do you think we should add it?

File: scripts/cargo-verify-seahorn.sh [hunk 3]
```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
```
Comment (alastairreid) [inline scripts/cargo-verify-seahorn.sh:1]: Is this script used instead of cargo-verify or is it called from cargo-verify?
(Or maybe something else?)

Comment (alastairreid) [inline scripts/cargo-verify-seahorn.sh:1]: Can you add the copyright/license boilerplate to this file?

**Review by alastairreid — APPROVED** [review]
Approving this on basis that there is more to come.

**Comment by fshaked** [comment]
Fixed everything mentioned above.

**Review by alastairreid — COMMENTED** [review]
LGTM

**Pull request merged**